catalog/core/catalog-core-solr/src/test/resources/simplelogger.properties

```diff
@@ -0,0 +1 @@
+org.slf4j.simpleLogger.defaultLogLevel=WARN
```

Comment:
This is to avoid a Windows surefire issue where too many logs are being printed.

Reply:
❓ Should we conditionally do this only for Windows?

Reply:
See previous comment